### PR TITLE
adding new kafka servers with vol encryption

### PR DESCRIPTION
**Pull request by prem-fissionhq**
https://dimagi-dev.atlassian.net/browse/SAAS-13161
##### ENVIRONMENTS AFFECTED

production

### CODE REVIEW

File: environments/production/terraform.yml [hunk 1]
```diff
@@ -293,6 +293,33 @@ servers:
...
+    os: bionic
+
+  - server_name: "kafka_b1-production"
+    server_instance_type: t3.medium
+    network_tier: "db-private"
+    az: "b"
+    volume_size: 300
+    volume_encrypted: yes
+    group: "kafka"
+    os: ubuntu_pro_bionic
```
Comment (dannyroberts) [inline environments/production/terraform.yml:312]: can use `bionic` for these two as well

Comment (prem-fissionhq) [reply environments/production/terraform.yml]: Done. Thanks Danny.

**Review by dannyroberts — APPROVED** [review]

**Comment by prem-fissionhq** [comment]
Created new PR. https://github.com/dimagi/commcare-cloud/pull/5285

**Comment by prem-fissionhq** [comment]
Created new PR. https://github.com/dimagi/commcare-cloud/pull/5285

**Pull request closed without merging**